x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_query_bar/exploration_query_bar.tsx

```diff
@@ -59,6 +60,31 @@ export const ExplorationQueryBar: FC<ExplorationQueryBarProps> = ({
 
   const searchChangeHandler = (q: Query) => setSearchInput(q);
 
+  const regex = useMemo(() => new RegExp(`${filters?.columnId}\s?:\s?(true|false)`, 'g'), []);
+
+  /**
+   * If a filter option is active in the url set the corresponding options
+   * button to selected mode
+   */
+  useEffect(() => {
```

Comment:
Let's use a named function instead. It makes troubleshooting easier, as the effect callback name appears in stack trace. 
```suggestion
  useEffect(function updateIdToSelectedMap() {
```

Reply:
Updated in a349738dc5dddfb51907e5d04aa22126df55ff9e